sdk/src/main/java/io/opentelemetry/sdk/trace/ReadableSpan.java

```diff
@@ -50,4 +51,12 @@
    * @since 0.1.0
    */
   SpanData toSpanData();
+
+  /**
+   * Returns the instrumentation library specified when creating the tracer which produced this
+   * span.
+   *
+   * @return a resource describing the instrumentation library
+   */
+  Resource getLibraryResource();
```

Comment:
does the interface need to expose this?

Reply:
Yes, the [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/6e8fdd8da816b21772ac0d59e59d1ecd038f907f/specification/sdk-tracing.md#L139-L141) says:

> The **readable representations** of all `Span` instances created by a `Tracer` must
provide a `getLibraryResource` method that returns this `Resource` information
held by the `Tracer`.

By exposing it on `ReadableSpan`, `SpanProcessor`s can access the library resource.

Reply:
➕